src/com/clover/remote/client/CloverConnector.ts

```diff
@@ -224,6 +224,8 @@ export class CloverConnector implements sdk.remotepay.ICloverConnector {
                 }
                 if (req.getTipMode()) {
                     transactionSettings.setTipMode(CloverConnector.getV3TipModeFromRequestTipMode(req.getTipMode()));
+                } else if (req.getDisableTipOnScreen()) {
```

Comment:
Transfer the disableTipOnScreen setting from the request to the transaction settings.